src/main/java/appeng/me/GridConnection.java

```diff
@@ -170,6 +178,15 @@ public ForgeDirection getDirection( final IGridNode side )
 	@Override
 	public void destroy()
 	{
+        if (AEConfig.instance.debugPathFinding)
+        {
+            final String aCoordinates = sideA.getGridBlock().getLocation().toString();
+            final String bCoordinates = sideB.getGridBlock().getLocation().toString();
+            AELog.info( "Repath is triggered by destroying connection from [%s] to [%s]", aCoordinates, bCoordinates );
+            Exception ex = new Exception();
```

Comment:
Would be a good thing to write a custom method logger with DEBUG level and print format Thread.currentThread().getStackTrace();

Reply:
I hope this is a temporary code, but anyway, I didn't know/looked up more clean way to print call stack, so thanks for the tip

Reply:
Level is irrelevant here because there is a debug command switching these parts